recipe/meta.yaml

```diff
@@ -66,10 +76,74 @@ requirements:
 test:
   requires:
     - pocl  # [linux]
+    - pytest
+    - pytest-xdist
   imports:
     - simtk
     - simtk.openmm
   commands:
+{% if with_test_suite %}
```

Comment:
Can you use script files like `run_test.sh` and `run_test.bat`?

Reply:
Yep, eventually I will move to those. I am not done yet!